### PR TITLE
Assembly references are now alphabetically ordered.

### PR DESCRIPTION
**Pull request by rovinbhandari**
I used it for a large project and having an alphabetically ordered list of assembly references made my life easier.

### CODE REVIEW

File: AsmSpy/Native/FileInfoExtensions.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace AsmSpy.Native
 {
-    internal static class FileInfoExtansions
+    internal static class FileInfoExtensions
```
Comment (mikehadlow) [inline AsmSpy/Native/FileInfoExtensions.cs]: Thanks for noticing the typo!

File: AsmSpy/Program.cs [hunk 2]
```diff
@@ -87,22 +87,24 @@ public static void AnalyseAssemblies(DirectoryInfo directoryInfo, bool onlyConfl
             if (onlyConflicts)
                 Console.WriteLine("Detailing only conflicting assembly references.");
 
-            foreach (var assembly in assemblies)
+            foreach (var assemblyName in assemblies.Keys.OrderBy(asm => asm, StringComparer.OrdinalIgnoreCase))
```
Comment (mikehadlow) [inline AsmSpy/Program.cs]: Why not just write `var assembly in assemblies.OrderBy(x => x.Key, StringComparer.OrdinalIgnoreCase)`
Then you wouldn't have to change the following lines.

Comment (rovinbhandari) [inline AsmSpy/Program.cs]: Resolved.

**Comment by mikehadlow** [comment]
Rovin, this is a really nice and useful contribution. Many thanks!

**Comment by rovinbhandari** [comment]
Happy to contribute! :smiley:

**Pull request merged**